egs/wsj/s5/steps/nnet3/xconfig_to_configs.py

```diff
@@ -170,7 +170,24 @@ def write_config_files(config_dir, all_layers):
             # preserves the backtrace
             raise
 
+    # remove previous init.config
+    try:
+        os.remove(config_dir + '/init.config')
+    except OSError:
+        pass
+
     for basename, lines in config_basename_to_lines.items():
+        # check the lines num start with 'output-node':
+        num_output_node_lines = sum( [ 1 if line.startswith('output-node' ) else 0
+                                       for line in lines ] )
+        if num_output_node_lines == 0:
```

Comment:
hi, @danpovey Check it again pls. 

Due to the output-node could > 1 in chain, I just check the 'num_output_node_lines == 0' condition.